### ISSUE TITLE
[INFO] Is it possible to use only icons in ActionDialogue?

### ISSUE THREAD

**Issue by kolliLakshmaiah1219** [oqtane/oqtane.framework#4586]
Is it possible to use only icons in ActionDialogue?

**Comment by sbwalker** [comment]
@kolliLakshmaiah1219 there is a parameter of IconOnly ie. ```<ActionDialog IconOnly="true" ...```

**Comment by sbwalker** [comment]
@mdmontesinos you added the enhancement for IconOnly in #4219 - can you please review this issue to determine if the component is working as expected or not?

**Comment by kolliLakshmaiah1219** [comment]
how do i create stored procedures in oqtane ,can you provide any links for that

**Comment by mdmontesinos** [comment]
@sbwalker It seems that my commit (#4219) broke localization and you fixed it back in (https://github.com/oqtane/oqtane.framework/commit/3648f999204a6748d186ac476cc20ebe0952d768). However, this broke the IconOnly implementation and I did not realize, as you set the _openText variable after it should have been cleared if IconOnly was true.

I just submitted a PR (https://github.com/oqtane/oqtane.framework/pull/4593) that handles both the localization and IconOnly.
